modules/nodes-pods-vertical-autoscaler-about.adoc

```diff
@@ -5,15 +5,15 @@
 [id="nodes-pods-vertical-autoscaler-about_{context}"]
 = About the Vertical Pod Autoscaler Operator
 
-The Vertical Pod Autoscaler Operator (VPA) is implemented as an API resource and a custom resource (CR). The CR determines the actions the Vertical Pod Autoscaler Operator should take with the pods associated with a specific workload object, such as a daemon set, replication controller, and so forth.
+The Vertical Pod Autoscaler Operator (VPA) is implemented as an API resource and a custom resource (CR). The CR determines the actions the Vertical Pod Autoscaler Operator should take with the pods associated with a specific workload object, such as a daemon set, replication controller, and so forth, in a project.
 
-The VPA automatically computes historic and current CPU and memory usage for the containers in those pods and can use this data to automatically re-deploy pods with optimized resource limits and requests to ensure that these pods are operating efficiently at all times. When re-deploying pods, the VPA honors any Pod Disruption Budget set for applications. If you do not want the VPA to automatically re-deploy pods, you can use this resource information to manually update the pods as needed.
+The VPA automatically computes historic and current CPU and memory usage for the containers in those pods and uses this data to determine optimized resource limits and requests to ensure that these pods are operating efficiently at all times. For example, the VPA reduces resources for pods that are requesting more resources than they are using and increases resources for pods that are not requesting enough.
 
-When configured to update pods automatically, the VPA reduces resources for pods that are requesting more resources then they are using and increase resources for pods that are not requesting enough.
+The VPA automatically deletes any pods that are out of alignment with its recommendations, one at a time so that your applications can continue to serve requests with no downtime. The workload objects then re-deploy the pods with the original resource limits and requests. The VPA uses a mutating admission webhook to update the pods with optimized resource limits and requests before the pods are admitted to a node. If you do not want the VPA to delete pods, you can view the VPA resource limits and requests and manually update the pods as needed.
```

Comment:
Throughout the PR: I think it should be "redeploy" (no `-`) since adding "re" isn't creating confusion with another word (like "recreate" vs "re-create" as mentioned in IBM SG). That's always what I'm seeing much more often in the rest of the repo.